docs/containers-storage.conf.5.md

```diff
@@ -34,6 +34,20 @@ The `storage` table supports the following options:
   container storage graph dir (default: "/var/lib/containers/storage")
   Default directory to store all writable content created by container storage programs.
 
+**rootless_storage_path**="$HOME/.local/share/containers/storage"
+  Storage path for rootless users. By default the graphroot for rootless users
+is set to `$XDG_DATA_HOME/containers/storage`, if XDG_DATA_HOME is set.
+Otherwise the `$HOME/.local/share/containers/storage` is used.  This field can
+be used if administrators need to change the storage location for all users.
+
+    The rootless storage path supports three substitions:
+    * `$HOME` => Replaced by the users home directory.
+    * `$UID`  => Replaced by the users UID
+    * `$USER` => Replace by the users name
+
+  A common use case for this field is `NFS home directories`, which is not
+supported for container storage on rootless users.
```

Comment:
"on rootless" -> "for rootless", or perhaps end with "container storage in a rootles environment."

Reply:
I wrote
```  A common use case for this field is `NFS home directories`, which do not work
with rootless container storage.
```